espnet2/bin/enh_scoring.py

```diff
@@ -64,6 +66,41 @@ def scoring(
         format="%(asctime)s (%(module)s:%(lineno)d) %(levelname)s: %(message)s",
     )
 
+    if dnsmos:
```

Comment:
This is a good feature. Can you add pesq as well?

Reply:
Yes, I can add the code. Please review whether we will have the license issue.
> The [`pesq`](https://github.com/ludlows/PESQ) python package seems to violate the PESQ license.